### ISSUE TITLE
Splitter: Missing Audible Labels

### ISSUE THREAD

**Issue by tmcconechy** [infor-design/enterprise#4404]
**Describe the bug**
The splitter collapse button and splitter handler are missing an audible label.

**To Reproduce**
1. Go to https://master-enterprise.demo.design.infor.com/components/splitter/test-collapse-button-right.html
2. Inspect the splitter collapse button / also the splitter handle
3. See error - its missing an audible span

**Expected behavior**
Add a translated/audible span.

**Version**
- ids-enterprise: [e.g. v4.32]

**Platform**
 - Chrome/Any

**Additional context**
https://jira.lawson.com/browse/LMCLIENT-31955

**Comment by CindyMercadoReyes** [comment]
This issue is now resolved.
